src/encord_active/lib/encord/actions.py

```diff
@@ -157,71 +225,74 @@ def create_project(
         all_new_label_rows = new_project.label_rows
         for counter, new_label_row in enumerate(all_new_label_rows):
```

Comment:
I know this isn't your code, but can we please make this less confusing?

 ```suggestion
         for counter, new_label_row_meta in enumerate(new_project.label_rows):
```

Reply:
btw, we can remove `all_new_label_rows` even tough it is used at a later point again because `new_project.label_rows` caches results.

Reply:
Lets do that at a later point when I understand it a bit better?